text/rustconf-steering-committee.md

```diff
@@ -5,46 +5,49 @@
+>  - **UnConf Day:** The Foundation would be pleased to offer an UnConf Day again in 2024, but would lean on the Steering Committee to decide this. If they decide there should be an UnConf Day after RustConf, the Foundation would like the Steering Committee to provide us with a basic plan including content and identification of people from the Project we should be logistically coordinating with ("UnConf Day Organizers") to ensure proper supplies and resources are available to UnConf organizers and attendees. The Foundation would welcome the Steering Committee's participation in planning and running the UnConf alongside the Organizers they identify.
+> - **Other considerations:** The Foundation would also like to lean on the Steering Committee to understand if there are any other areas of the event they would like to be involved in on behalf of the Rust Project. The Rust Foundation needs to operate independently on major event decisions (like co-located events and conference activities, tickets/access, general event experience, and sponsorship experience) but will welcome feedback from the Steering Committee along the way.
 
-Once a steering committee has been formed, we can establish a regular process of communication between the Rust Foundation, the steering committee, and other groups from within the Project as identified by the steering committee. 
+- **Communication:** The Rust Foundation will share monthly asynchronous updates with the Steering Committee as the RustConf planning process develops. These updates will be used to share advanced notice of key event milestones (i.e. CFP open/close date, registration details, major planning changes/developments). The steering committee can use an agreed-upon communication channel to ask questions/provide feedback about the details shared. 
 
 
 # Motivation
 
-While the Rust Foundation has served as a Diamond-level sponsor of RustConf since our inception in 2021, we have sought approval to use our staff and financial resources towards planning and managing RustConf in a larger way in 2024. We have gotten approval from our board and received [encouragement](https://rust-lang.zulipchat.com/#narrow/stream/335408-foundation/topic/RustConf.202024.20next.20steps/near/403266679) from the Project and Leadership Council to proceed in Q4.
+Since it was approved by the board and leadership council for the Rust Foundation to play a larger role in managing RustConf, we have intended to facilitate an UnConf Day and for the Program Committee to remain independent (with management and facilitation provided by the Foundation). We also understand that there may be additional expectations around Rust Project involvement and feel that clearer parameters around the Rust Project's role in RustConf would be helpful.
```

Comment:
```suggestion
Since it was approved by the Rust Foundation Board and leadership council for the Rust Foundation to play a larger role in managing RustConf, we have intended to facilitate an UnConf Day and for the Program Committee to remain independent (with management and facilitation provided by the Foundation). We also understand that there may be additional expectations around Rust Project involvement and feel that clearer parameters around the Rust Project's role in RustConf would be helpful.
```